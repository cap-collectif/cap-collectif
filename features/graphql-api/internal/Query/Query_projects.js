//* eslint-env jest */
const ProjectQuery = /* GraphQL */ `
    query ProjectQuery(
        $count: Int
        $cursor: String
        $archived: ProjectArchiveFilter
        $status: ID
      ) {
        projects(first: $count, after: $cursor, archived: $archived, status: $status) {
            totalCount
            edges {
                node {
                    id
                    title
                    archived
                    status
                    __typename
                }
                cursor
            }
        }
    }
`;

const variables = {
  "count": 16,
  "cursor": null,
  "archived": null,
  "status": null
}

describe('Internal|Query projects', () => {

  it('fetches all projects', async () => {
    await expect(graphql(ProjectQuery, variables, 'internal_admin')).resolves.toMatchSnapshot();
  });

  it('fetches archived projects', async () => {
    await expect(graphql(ProjectQuery, {
      ...variables,
      archived: "ARCHIVED"
    }, 'internal_admin')).resolves.toMatchSnapshot();
  });

  it('fetches non archived projects', async () => {
    await expect(graphql(ProjectQuery, {
      ...variables,
      archived: "ACTIVE"
    }, 'internal_admin')).resolves.toMatchSnapshot();
  });

  it('fetches opened projects', async () => {
    await expect(graphql(ProjectQuery, {
      ...variables,
      status: 1,
    }, 'internal_admin')).resolves.toMatchSnapshot();
  });

  it('fetches opened participation projects', async () => {
    await expect(graphql(ProjectQuery, {
      ...variables,
     status: 2,
    }, 'internal_admin')).resolves.toMatchSnapshot();
  });

  it('fetches closed projects', async () => {
    await expect(graphql(ProjectQuery, {
      ...variables,
      status: 3,
    }, 'internal_admin')).resolves.toMatchSnapshot();
  });

});
