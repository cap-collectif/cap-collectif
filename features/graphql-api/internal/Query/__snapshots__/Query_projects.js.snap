// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Internal|Query projects fetches all projects 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dDb2xsZWN0Vm90ZQ==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote étape de dépôt",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3RlMg==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours min 2 max 2",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0UHJvcG9zYWxBcmNoaXZpbmc=",
          "status": "OPENED_PARTICIPATION",
          "title": "BP avec archivage des propositions",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE9ubHlPdGhlclN0ZXBPcGVu",
          "status": "OPENED",
          "title": "BP avec étape personalisée ouverte",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0RGViYXRlT3BlbldpdGhvdXRWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "Débats ouvert sans vote",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjY=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T3JnYVZpc2liaWxpdHlBZG1pbkFuZE1l",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif d'organisation",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjc=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T3JnYVZpc2liaWxpdHlNZQ==",
          "status": "UNKNOWN",
          "title": "projectOrgaVisibilityMe",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjg=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpyYXBwb3J0RXZhbHVhdGlvbg==",
          "status": "OPENED_PARTICIPATION",
          "title": "Rapport d'évaluation 6 (AR6)",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjk=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0SWRmMw==",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif IdF 3",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEw",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Rm9vZA==",
          "status": "OPENED_PARTICIPATION",
          "title": "Food project",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEx",
        "node": Object {
          "__typename": "Project",
          "archived": true,
          "id": "UHJvamVjdDpwcm9qZWN0QXJjaGl2ZWQ=",
          "status": "UNKNOWN",
          "title": "Projet Archivé",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEy",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q29uZmluZW1lbnQ=",
          "status": "CLOSED",
          "title": "Débat du mois",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEz",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aEFub255bW91c1F1ZXN0aW9ubmFpcmU=",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec questionnaire anonyme",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE0",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE93bmVy",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec administrateur de projet",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE1",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V3lzaXd5Zw==",
          "status": "CLOSED",
          "title": "Débat Wysiwyg",
        },
      },
    ],
    "totalCount": 51,
  },
}
`;

exports[`Internal|Query projects fetches archived projects 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "node": Object {
          "__typename": "Project",
          "archived": true,
          "id": "UHJvamVjdDpwcm9qZWN0QXJjaGl2ZWQ=",
          "status": "UNKNOWN",
          "title": "Projet Archivé",
        },
      },
    ],
    "totalCount": 1,
  },
}
`;

exports[`Internal|Query projects fetches closed projects 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q29uZmluZW1lbnQ=",
          "status": "CLOSED",
          "title": "Débat du mois",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V3lzaXd5Zw==",
          "status": "CLOSED",
          "title": "Débat Wysiwyg",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0TXVsdGlEZWJhdGU=",
          "status": "CLOSED",
          "title": "Débats sur la vie",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q29yb25h",
          "status": "CLOSED",
          "title": "Solidarité Covid-19",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Mw==",
          "status": "CLOSED",
          "title": "Transformation numérique des relations",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0R3JvdXAy",
          "status": "CLOSED",
          "title": "Projet pour le group2",
        },
      },
    ],
    "totalCount": 6,
  },
}
`;

exports[`Internal|Query projects fetches non archived projects 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dDb2xsZWN0Vm90ZQ==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote étape de dépôt",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3RlMg==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours min 2 max 2",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0UHJvcG9zYWxBcmNoaXZpbmc=",
          "status": "OPENED_PARTICIPATION",
          "title": "BP avec archivage des propositions",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE9ubHlPdGhlclN0ZXBPcGVu",
          "status": "OPENED",
          "title": "BP avec étape personalisée ouverte",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0RGViYXRlT3BlbldpdGhvdXRWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "Débats ouvert sans vote",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjY=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T3JnYVZpc2liaWxpdHlBZG1pbkFuZE1l",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif d'organisation",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjc=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T3JnYVZpc2liaWxpdHlNZQ==",
          "status": "UNKNOWN",
          "title": "projectOrgaVisibilityMe",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjg=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpyYXBwb3J0RXZhbHVhdGlvbg==",
          "status": "OPENED_PARTICIPATION",
          "title": "Rapport d'évaluation 6 (AR6)",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjk=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0SWRmMw==",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif IdF 3",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEw",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Rm9vZA==",
          "status": "OPENED_PARTICIPATION",
          "title": "Food project",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEx",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q29uZmluZW1lbnQ=",
          "status": "CLOSED",
          "title": "Débat du mois",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEy",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aEFub255bW91c1F1ZXN0aW9ubmFpcmU=",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec questionnaire anonyme",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEz",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE93bmVy",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec administrateur de projet",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE0",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V3lzaXd5Zw==",
          "status": "CLOSED",
          "title": "Débat Wysiwyg",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE1",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T2N1bHVzUXVlc3Q=",
          "status": "FUTURE_WITHOUT_FINISHED_STEPS",
          "title": "Débat sur l'Oculus Quest",
        },
      },
    ],
    "totalCount": 50,
  },
}
`;

exports[`Internal|Query projects fetches opened participation projects 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dDb2xsZWN0Vm90ZQ==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote étape de dépôt",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3RlMg==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours min 2 max 2",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0UHJvcG9zYWxBcmNoaXZpbmc=",
          "status": "OPENED_PARTICIPATION",
          "title": "BP avec archivage des propositions",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0RGViYXRlT3BlbldpdGhvdXRWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "Débats ouvert sans vote",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T3JnYVZpc2liaWxpdHlBZG1pbkFuZE1l",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif d'organisation",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjY=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpyYXBwb3J0RXZhbHVhdGlvbg==",
          "status": "OPENED_PARTICIPATION",
          "title": "Rapport d'évaluation 6 (AR6)",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjc=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0SWRmMw==",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif IdF 3",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjg=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Rm9vZA==",
          "status": "OPENED_PARTICIPATION",
          "title": "Food project",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjk=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aEFub255bW91c1F1ZXN0aW9ubmFpcmU=",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec questionnaire anonyme",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEw",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE93bmVy",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec administrateur de projet",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEx",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q2FmZXRpZXI=",
          "status": "OPENED_PARTICIPATION",
          "title": "Sauvons nos cafés",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEy",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q2FubmFiaXM=",
          "status": "OPENED_PARTICIPATION",
          "title": "Débat sur le cannabis",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEz",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0RXhwb3J0",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet pour consolider les exports",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE0",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0SWRm",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif IdF",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE1",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0QW5hbHlzZQ==",
          "status": "OPENED_PARTICIPATION",
          "title": "Project Analyse",
        },
      },
    ],
    "totalCount": 36,
  },
}
`;

exports[`Internal|Query projects fetches opened projects 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjA=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dDb2xsZWN0Vm90ZQ==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote étape de dépôt",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjI=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V29ya2Zsb3dWb3RlMg==",
          "status": "OPENED_PARTICIPATION",
          "title": "BP vote parcours min 2 max 2",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjM=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0UHJvcG9zYWxBcmNoaXZpbmc=",
          "status": "OPENED_PARTICIPATION",
          "title": "BP avec archivage des propositions",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjQ=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE9ubHlPdGhlclN0ZXBPcGVu",
          "status": "OPENED",
          "title": "BP avec étape personalisée ouverte",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjU=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0RGViYXRlT3BlbldpdGhvdXRWb3Rl",
          "status": "OPENED_PARTICIPATION",
          "title": "Débats ouvert sans vote",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjY=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0T3JnYVZpc2liaWxpdHlBZG1pbkFuZE1l",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif d'organisation",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjc=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpyYXBwb3J0RXZhbHVhdGlvbg==",
          "status": "OPENED_PARTICIPATION",
          "title": "Rapport d'évaluation 6 (AR6)",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjg=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0SWRmMw==",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif IdF 3",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjk=",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Rm9vZA==",
          "status": "OPENED_PARTICIPATION",
          "title": "Food project",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEw",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aEFub255bW91c1F1ZXN0aW9ubmFpcmU=",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec questionnaire anonyme",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEx",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0V2l0aE93bmVy",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet avec administrateur de projet",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEy",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q2FmZXRpZXI=",
          "status": "OPENED_PARTICIPATION",
          "title": "Sauvons nos cafés",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjEz",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0Q2FubmFiaXM=",
          "status": "OPENED_PARTICIPATION",
          "title": "Débat sur le cannabis",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE0",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0RXhwb3J0",
          "status": "OPENED_PARTICIPATION",
          "title": "Projet pour consolider les exports",
        },
      },
      Object {
        "cursor": "YXJyYXljb25uZWN0aW9uOjE1",
        "node": Object {
          "__typename": "Project",
          "archived": false,
          "id": "UHJvamVjdDpwcm9qZWN0SWRm",
          "status": "OPENED_PARTICIPATION",
          "title": "Budget Participatif IdF",
        },
      },
    ],
    "totalCount": 37,
  },
}
`;

exports[`Internal|Query projects search by project creator username 1`] = `
Object {
  "viewer": Object {
    "projects": Object {
      "edges": Array [
        Object {
          "node": Object {
            "creator": Object {
              "username": "welcomattic",
            },
            "title": "Budget Participatif Rennes",
          },
        },
        Object {
          "node": Object {
            "creator": Object {
              "username": "welcomattic",
            },
            "title": "Projet avec questionnaire",
          },
        },
        Object {
          "node": Object {
            "creator": Object {
              "username": "welcomattic",
            },
            "title": "Projet externe",
          },
        },
      ],
    },
  },
}
`;

exports[`Internal|Query projects search by project organization creator username 1`] = `
Object {
  "viewer": Object {
    "projects": Object {
      "edges": Array [
        Object {
          "node": Object {
            "creator": Object {
              "username": "welcomattic",
            },
            "title": "Budget Participatif Rennes",
          },
        },
        Object {
          "node": Object {
            "creator": Object {
              "username": "welcomattic",
            },
            "title": "Projet avec questionnaire",
          },
        },
        Object {
          "node": Object {
            "creator": Object {
              "username": "welcomattic",
            },
            "title": "Projet externe",
          },
        },
      ],
    },
  },
}
`;

exports[`Internal|Query projects search by project organization owner username 1`] = `
Object {
  "viewer": Object {
    "projects": Object {
      "edges": Array [
        Object {
          "node": Object {
            "owner": Object {
              "username": "GIEC",
            },
            "title": "Budget Participatif d'organisation",
          },
        },
        Object {
          "node": Object {
            "owner": Object {
              "username": "GIEC",
            },
            "title": "projectOrgaVisibilityMe",
          },
        },
        Object {
          "node": Object {
            "owner": Object {
              "username": "GIEC",
            },
            "title": "Rapport d'évaluation 6 (AR6)",
          },
        },
      ],
    },
  },
}
`;

exports[`Internal|Query projects search by project owner username 1`] = `
Object {
  "viewer": Object {
    "projects": Object {
      "edges": Array [
        Object {
          "node": Object {
            "owner": Object {
              "username": "GIEC",
            },
            "title": "Budget Participatif d'organisation",
          },
        },
        Object {
          "node": Object {
            "owner": Object {
              "username": "GIEC",
            },
            "title": "projectOrgaVisibilityMe",
          },
        },
        Object {
          "node": Object {
            "owner": Object {
              "username": "GIEC",
            },
            "title": "Rapport d'évaluation 6 (AR6)",
          },
        },
      ],
    },
  },
}
`;

exports[`Internal|Query projects search by project title 1`] = `
Object {
  "projects": Object {
    "edges": Array [
      Object {
        "node": Object {
          "title": "BP avec archivage des propositions",
        },
      },
    ],
  },
}
`;
